filesystem.go

```diff
@@ -173,11 +168,6 @@ func (fs *filesystem) storeFile(path string, data interface{}) error {
 }
 
 func (fs *filesystem) fetchFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
Why do you want to remove this one ? You have to return an error in case you cannot find the file which is supposed to be stored, don't you ?